shell/src/main/java/org/apache/accumulo/shell/commands/ListScansCommand.java

```diff
@@ -75,14 +77,31 @@ public int numArgs() {
   public Options getOptions() {
     final Options opts = new Options();
 
-    tserverOption = new Option("ts", "tabletServer", true, "tablet server to list scans for");
-    tserverOption.setArgName("tablet server");
-    opts.addOption(tserverOption);
+    serverOpt = new Option("s", "server", true, "tablet/scan server regex to list scans for");
```

Comment:
In #4958 I left the `ts` parameter and made it work for all types. I also added a `s` parameter which does the same thing. My thinking was that then users would not have to fix any scripts that they have written. I would suggest that we have a common approach to this, so either you do the same here, or we remove `ts` from PingCommand.

Reply:
Sounds good, I can just go with the same approach already taken in #4958 so we are consistent